cisco_sdwan/assets/monitors/tunnel_down.json

```diff
@@ -0,0 +1,31 @@
+{
+	"version": 2,
+	"created_at": "2024-04-05",
+	"last_updated_at": "2024-04-05",
+	"title": "Cisco SD-WAN Tunnel is down",
+	"tags": [
+		"integration:cisco-sdwan"
+	],
+	"description": "This monitor detects when a Cisco SD-WAN tunnel becomes unavailable between two devices. It tracks the status of tunnels within a specified namespace and triggers an alert if a tunnel goes down, indicating potential connectivity issues.",
+	"definition": {
+		"name": "[Cisco SD-WAN] Tunnel is down between {{local_color.name}} {{hostname.name}} ({{system_ip.name}}) to {{remote_color.name}} {{remote_hostname.name}} ({{remote_system_ip.name}})",
+		"type": "query alert",
+		"query": "min(last_15m):min:cisco_sdwan.tunnel.status{*} by {device_namespace,system_ip,hostname,local_color,remote_system_ip,remote_hostname,remote_color} < 1",
+		"message": "{{#is_alert}}\nSD-WAN Tunnel is down between device {{hostname.name}} ({{system_ip.name}}), color {{local_color.name}} to device {{remote_hostname.name}} ({{remote_system_ip.name}}), color {{remote_color.name}}.\n{{/is_alert}}\n\n{{#is_alert_recovery}}\nSD-WAN Tunnel is back up between device {{hostname.name}} ({{system_ip.name}}), color {{local_color.name}} to device {{remote_hostname.name}} ({{remote_system_ip.name}}), color {{remote_color.name}}.\n{{/is_alert_recovery}}\n\nTo know more about the status of your device, you can have more information from the [NDM page for the device {{device_namespace.name}}:{{system_ip.name}}](/infrastructure/devices/graph?inspectedDevice={{device_namespace.name}}%3A{{system_ip.name}}).",
+		"tags": [],
+		"options": {
+			"thresholds": {
+				"critical": 1
+			},
+			"notify_audit": false,
+			"include_tags": true,
+			"new_group_delay": 60,
+			"notify_no_data": false,
+			"silenced": {},
+			"avalanche_window": 10,
+			"evaluation_delay": 1200
```

Comment:
That's a good call out, the evaluation_delay is required for metrics that are known to be delayed.
Should we use it for other monitors too? How did you come up with 1200

Reply:
Other monitors are set on metrics that are not "delayed" (see [device reboots](https://github.com/DataDog/datadog-agent/blob/c8d4425e5094ac2cfe51b42f879140295fb58858/pkg/collector/corechecks/network-devices/cisco-sdwan/report/metrics.go#L225) and [device status](https://github.com/DataDog/datadog-agent/blob/c8d4425e5094ac2cfe51b42f879140295fb58858/pkg/collector/corechecks/network-devices/cisco-sdwan/report/metrics.go#L233))
1200 (20 minutes) is the usual maximum delay when polling the statistics interval (there's no official documentation about this though 🙁)